CorsixTH/Src/th_gfx_sdl.cpp

```diff
@@ -26,19 +26,23 @@ SOFTWARE.
 #ifdef CORSIX_TH_USE_FREETYPE2
 #include "th_gfx_font.h"
 #endif
+#include <SDL.h>  // IWYU pragma: keep
```

Comment:
The keep on SDL.h isn't particularly helpful here or the other spot I put it, drop them.